dracut.sh

```diff
@@ -1093,6 +1093,13 @@ drivers_dir="${drivers_dir%"${drivers_dir##*[!/]}"}"
 [[ $sbat_l ]] && sbat="$sbat_l"
 [[ $machine_id_l ]] && machine_id="$machine_id_l"
 
+if ! [[ $outfile ]] && [[ $initrdname ]]; then
```

Comment:
The condition `if [[ $initrdname ]]` should be moved within the next `if ! [[ $outfile ]]`, i.e., if `outfile` has any value, `initrdname` is useless.